leo/commands/lint.rs

```diff
@@ -14,65 +14,35 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    cli::*,
-    cli_types::*,
-    commands::BuildCommand,
-    errors::{CLIError, RunError},
-};
-use leo_package::{
-    root::Manifest,
-    source::{MAIN_FILENAME, SOURCE_DIRECTORY_NAME},
-};
-
-use clap::ArgMatches;
-use std::{convert::TryFrom, env::current_dir};
-
-#[derive(Debug)]
-pub struct LintCommand;
+use crate::{commands::Command, context::Context};
+use anyhow::Result;
```

Comment:
nit: please organize the imports here into two sections - leo specific crates & external crates